Cargo.toml

```diff
@@ -87,7 +87,7 @@ regex = "1.3"
 serde = { version = "1.0", features=["derive"] }
 serde_json = "1.0"
 serde_urlencoded = "0.6.1"
-time = "0.1.42"
+time = "0.2.2"
```

Comment:
If you're not using any deprecated features, you can do `time = { version = "0.2.2", default-features = false, features = ["std"] }`. It you are using any deprecated methods, using this will cause compilation to fail, so it should be easy to check.

Reply:
I avoided all deprecated features while making the initial changes to not need to remove them later, but I must've forgot about changing the package definition to that (or thought it was disabled by default) - changed all instances of time in the various Cargo.tomls to that string.